node/src/chaincodeeventsbuilder.ts

```diff
@@ -10,12 +10,7 @@ import { EventsBuilder, EventsOptions } from './eventsbuilder';
 import { ChaincodeEventsRequest as ChaincodeEventsRequestProto } from './protos/gateway/gateway_pb';
 import { SigningIdentity } from './signingidentity';
 
-/**
- * Options used when requesting chaincode events.
- */
-export type ChaincodeEventsOptions = EventsOptions;
-
-export interface ChaincodeEventsBuilderOptions extends ChaincodeEventsOptions {
+export interface ChaincodeEventsBuilderOptions extends EventsOptions {
```

Comment:
`ChaincodeEventsOptions` was supposed to still be here for backwards compatibility after I added the common `EventsOptions` for use by both chaincode and block eventing